library/src/tests/resources/add_le.qs

```diff
@@ -25,12 +26,9 @@ namespace Test {
                 let zActual = MeasureInteger(z);
                 let zExpected = (xValue + yValue) % (1 <<< zLen);
 
-                Fact(xActual == xValue,
-                   $"{name}: Incorrect x={xActual}, expected={xValue}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}.");
-                Fact(yActual == yValue,
-                   $"{name}: Incorrect y={yActual}, expected={yValue}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}.");
-                Fact(zActual == zExpected,
-                   $"{name}: Incorrect z={zActual}, expected={zExpected}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}.");
+                Fact(xActual == xValue, $"{name}: Incorrect x={xActual}, expected={xValue}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}.");
+                Fact(yActual == yValue, $"{name}: Incorrect y={yActual}, expected={yValue}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}.");
+                Fact(zActual == zExpected, $"{name}: Incorrect z={zActual}, expected={zExpected}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}.");
```

Comment:
This line is fairly long (~160 characters). Is there a limit after which we try to split long lines?

Reply:
Line length limits are one of the things this formatter can't do. Since we are removing user's ability to put newlines where they want with this PR (that's the point of this PR), some lines may become longer than desired. In these cases, users can either write their Q# in a way that avoids the long lines, or do not format that section of their code.

Reply:
But we are formatting entire library now as a policy. Long lines don't look good.

Reply:
But hang on... this particular line _can_ be improved by formatting the argument list vertically, right? I think we should just do that.

```qs
Fact(
    xActual == xValue,
    $"{name}: Incorrect x={xActual}, expected={xValue}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}."
);
Fact(
    yActual == yValue,
    $"{name}: Incorrect y={yActual}, expected={yValue}. |x|={xLen}, |y|={yLen}, |z|={zLen}, x={xValue}, y={yValue}."
);
```

The only "problem" here is that the formatter won't do this automatically for you based on line length - you have to add the first newline manually. But the formatter respects your choice after that.

Reply:
That does look better like that. Thanks Mine, I've applied your suggestion.